### PR TITLE
Make v3 changes

### PR DESCRIPTION
**Pull request by maysakanoni**

### CODE REVIEW

File: .github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml [hunk 1]
```diff
@@ -26,7 +27,7 @@ jobs:
           cache: 'yarn'
       - run: yarn install --frozen-lockfile
       - run: yarn lint
-      - run: yarn test
+#      - run: yarn test
```
Comment (elena-vi) [inline .github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml:30]: I think this should be restored

File: .github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml [hunk 2]
```diff
@@ -7,6 +7,7 @@ on:
   pull_request:
     types: [opened, synchronize, reopened, closed]
     branches:
+      - next.js
```
Comment (elena-vi) [inline .github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml:10]: why was this added?

Comment (maysakanoni) [reply .github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml:10]: this was to run the test when i create a pr off this branch, because it was previously only set to main

Comment (maysakanoni) [reply .github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml]: removed

File: .github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml (hunk 1 above)
Comment (maysakanoni) [reply .github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml]: resolved

**Comment by maysakanoni** [comment]
[Rework - ensure current MVP pages match the beta v3 prototype](https://trello.com/c/qkcTY9iK/193-rework-ensure-current-mvp-pages-match-the-beta-v3-prototype)

**Comment by maysakanoni** [comment]
[Rework - ensure current MVP pages match the beta v3 prototype](https://trello.com/c/qkcTY9iK/193-rework-ensure-current-mvp-pages-match-the-beta-v3-prototype)

File: pages/index.js [hunk 3]
```diff
@@ -12,33 +12,61 @@ export default function Home() {
...
-        <h1>Request a repair</h1>
-        <h2>Before you start</h2>
-        <WarningText testid="landing-page-gas-warning">
+        <h1 className={'govuk-heading-xl'}>Request a repair</h1>
+        <h2 className={'govuk-heading-m'}>Before you start</h2>
+        <WarningText  testid="landing-page-report-limit-warning" className="govuk-!-margin-top-4">
+          This service can only be used to request one repair at a time to a council property.
+        </WarningText>
+        <p>
+          To request a repair for a <strong>leaseholder</strong>, please see our <TextLink href="https://www.lincoln.gov.uk/council-housing/council-leaseholders/3">leaseholder contact details page </TextLink>.
```
Comment (Bhavesh-Patel) [inline pages/index.js:23]: This link should open in a new tab

File: pages/index.js [hunk 4]
```diff
@@ -12,33 +12,61 @@ export default function Home() {
...
+        </p>
+        <p>
+          To request a repair for a <strong>communal area</strong>, please see customer services contact <TextLink href="https://www.lincoln.gov.uk/contact-1/get-touch">details page</TextLink>.
```
Comment (Bhavesh-Patel) [inline pages/index.js:26]: This link should open in a new tab

File: pages/index.js [hunk 5]
```diff
@@ -12,33 +12,61 @@ export default function Home() {
...
+        <br/>
+        <WarningText testid="landing-page-gas-warning" className="govuk-!-margin-top-4">
             If you suspect you have a gas leak, you must report it immediately
             to the Gas Emergency Service on 0800 111 999 or via textphone
             (minicom) on 0800 371 787
         </WarningText>
-        <WarningText testid="landing-page-emergency-warning" className="govuk-!-margin-top-4">
-            For other emergency repairs, please see customer services {contactDetailsPageLink}.
+        <WarningText testid="landing-page-emergency-warning"  className="govuk-!-margin-top-4">
+          For other emergency repairs, please see customer services <TextLink href="https://www.lincoln.gov.uk/contact-1/get-touch">contact details page</TextLink>.
```
Comment (Bhavesh-Patel) [inline pages/index.js:53]: This link should open in a new tab

File: pages/index.js [hunk 6]
```diff
@@ -12,33 +12,61 @@ export default function Home() {
...
-                Emergency Out of Hours Repairs, please call <strong>01522 534747</strong>
+              Emergency Out of Hours Repairs, please call <strong>01522 534747</strong>
             </p>
             <p>
-                For emergencies in normal hours, please call <strong>01522 873333</strong>
+              For emergencies in normal hours, please call <strong>01522 873333</strong>
             </p>
             <p>
-                Opening times:
+              Opening times:
```
Comment (Bhavesh-Patel) [inline pages/index.js:69]: Opening hours in the prototype are different to the MVP:
Differences:

Prototype has Wednesday 10am - 4.30pm and MVP has Wednesday 9am - 4.30pm
Prototype has Thursday 9am - 5pm and MVP has Thursday 10am - 5pm

File: compoments/report-repair/emergency-repair.js [hunk 7]
```diff
@@ -3,18 +3,39 @@ import React from 'react';
...
+      </p>
+      <ul>
+        <li>Monday 9am - 5pm</li>
+        <li>Tuesday 9am - 5pm</li>
+        <li>Wednesday 10am - 4.30pm</li>
+        <li>Thursday 9am - 5pm</li>
+        <li>Friday 9am - 4.30pm</li>
+      </ul>
+      <p>
+        For non-emergency requests, call <strong>01522 873 333</strong>
```
Comment (Bhavesh-Patel) [inline compoments/report-repair/emergency-repair.js:30]: Please update telephone number format to 5-6 digits, i.e. 01522 873333
```suggestion
        For non-emergency requests, call <strong>01522 873333</strong>
```

File: compoments/report-repair/address.js [hunk 8]
```diff
@@ -43,39 +43,39 @@ const Address = ({handleChange, values}) => {
...
-            <Select
-              input={{
-                name: 'address',
-                onChange: onChange
-              }}
-              meta={state.error}
-            >
-              <option value="null">
-                {found_addresses}
+      <h1 className="govuk-heading-xl">Where is the repair located?</h1>
```
Comment (Bhavesh-Patel) [inline compoments/report-repair/address.js:46]: The title should be "Select an address"
```suggestion
      <h1 className="govuk-heading-xl">Select an address</h1>
```

**Review by Bhavesh-Patel — CHANGES_REQUESTED** [review]
A few textual/copy changes.
Added suggestions where I could.

**Review by Bhavesh-Patel — APPROVED** [review]
